src/main/java/com/exasol/adapter/installer/Installer.java

```diff
@@ -132,29 +132,26 @@ private String prepareVirtualSchemaStatement() {
         return createVirtualSchemaStatement.toString();
     }
 
-    private String prepareAdapterScriptStatement(final String schemaName, final String adapterName) {
-        return "CREATE OR REPLACE JAVA ADAPTER SCRIPT " + schemaName + "." + adapterName + " AS" + LINE_SEPARATOR //
-                + "  %scriptclass com.exasol.adapter.RequestDispatcher;" + LINE_SEPARATOR //
-                + "%jar /buckets/bfsdefault/" + this.exaBucketName + "/" + this.virtualSchemaJarName + ";"
-                + LINE_SEPARATOR //
-                + "%jar /buckets/bfsdefault/" + this.exaBucketName + "/" + this.jdbcDriverName + ";";
+    private String prepareAdapterScriptStatement(final List<String> jarsInBucket) {
+        final StringBuilder adapterScript = new StringBuilder( //
+                "CREATE OR REPLACE JAVA ADAPTER SCRIPT " + this.exaSchemaName + "." + this.exaAdapterName + " AS"
+                        + LINE_SEPARATOR //
+                        + "  %scriptclass com.exasol.adapter.RequestDispatcher;" + LINE_SEPARATOR //
+        );
+        for (final String jarName : jarsInBucket) {
+            adapterScript.append("%jar /buckets/bfsdefault/").append(this.exaBucketName).append("/").append(jarName)
+                    .append(";").append(LINE_SEPARATOR);
```

Comment:
can we extract this to a dedicated method?

maybe something like `adapterScript.append(getPathInBucketFor(jarFilename));` or something similar.